dependencies/Dockerfile

```
@@ -2,25 +1,62 @@
+COPY --chmod=0755 build_imagemagick.sh .
+RUN --mount=type=secret,id=github_token \
+    && export GITHUB_TOKEN=$(cat /run/secrets/github_token) \
+    && export $(cat /env) && export $(cat /versions.env) && ./build_imagemagick.sh
 
-COPY prepare.sh /tmp/build/
-RUN ./prepare.sh
+FROM base-prep AS download-jellyfin-ffmpeg
+ARG JELLYFIN_FFMPEG_VERSION
+WORKDIR /tmp/build-jellyfin
+COPY --chmod=0755 download_jellyfin-ffmpeg.sh .
+RUN --mount=type=secret,id=github_token \
```

Hadolint (2.12.0)

[error] Unexpected start of line. If breaking lines, |/||/&& should be at the end of the previous one.

(SC1133)